src/main/java/ch/ipt/see/playground/MathController.java

```diff
@@ -0,0 +1,54 @@
+package ch.ipt.see.playground;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+@RestController
+public class MathController {
+    private final DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+
+    @PostMapping(
+            path = "/math/area_computation/hyperrectangle",
+            consumes = MediaType.APPLICATION_XML_VALUE,
+            produces = MediaType.APPLICATION_XML_VALUE
+    )
+    public ResponseEntity<String> calculateRectangleArea(@RequestBody String input) {
```

Comment:
Vorschlag: umbenennen
```suggestion
    public ResponseEntity<String> calculateHyperrectangleVolume(@RequestBody String input) {
```